src/app/layout.tsx

```diff
@@ -33,6 +34,17 @@ export const metadata: Metadata = {
 export default function RootLayout({ children }: { children: React.ReactNode }) {
   return (
     <html lang="en">
+      <Script async src="https://www.googletagmanager.com/gtag/js?id=G-WY6K7SWSCE" />
```

Comment:
Let's use the built-in library
[https://developers.google.com/tag-platform/gtagjs](https://developers.google.com/tag-platform/gtagjs)

Reply:
It gave me an error like this.. 
> Prefer `next/script` component when using the inline script for Google Analytics. See: https://nextjs.org/docs/messages/next-script-for-gaeslint[@next/next/next-script-for-ga](https://nextjs.org/docs/messages/next-script-for-ga)